boards/riscv/rv32m1_vega/doc/index.rst

```diff
@@ -150,8 +151,7 @@ with this board:
 
    - **no 2 Mbps PHY**
    - no 512/256 Kbps PHY
-   - **no controller-based cryptographic services (neither RPA nor LL encryption
-     or decryption)**
+   - **no RPA**
```

Comment:
What exactly do you mean by this? That you don't have controller-side privacy? The host has support for it and will take over if the controller indicates in its features that it doesn't support it.

Reply:
Privacy is submitted in a separate PR. While Privacy could work without encryption if the IRKs are predetermined and stored in the flash, in general needs an encrypted link to exchange keys. These two features are not packed together in the same PR. So, each PR updates the documentation independently, modifying the above line without assuming the other PR was merged or not. During the bug-fixing window, we planned to correct this line to accurately reflect what is included in the release, if both were merged.

Reply:
We could put `no LL Privacy` instead perhaps? stating no RPA is a bit vague.